### PR TITLE
#20 - Search Teasers Slotting

### PR DESCRIPTION
**Pull request by hunyoboy**

### CODE REVIEW

**Comment by hunyoboy** [comment]
@bhsiao55 Can you take a look? Thanks.

File: src/SearchTeasersRequestHandler.php [hunk 1]
```diff
@@ -4,19 +4,214 @@
...
+    protected function getSlottedNodes(SearchNodesRequest $request, Pbjx $pbjx): array
+    {
+        if (!$request->has('slotting_key')
+            || !NodeStatus::PUBLISHED()->equals($request->get('status'))
+        ) {
+            return [];
+        }
+
+        $slottingKey = $request->get('slotting_key');
+        $cacheKey = "news.slotting.{$slottingKey}.php";
```
Comment (bhsiao55) [inline src/SearchTeasersRequestHandler.php]: this key should not have news

Comment (hunyoboy) [reply src/SearchTeasersRequestHandler.php]: good find @bhsiao55 
@gdbrown do we need a separate key from article search here?

Comment (gdbrown) [reply src/SearchTeasersRequestHandler.php]: @hunyoboy yes of course.  use `"curator.slotting.{$slottingKey}.php"`

**Pull request merged**